internal/metrics/registration.go

```diff
@@ -0,0 +1,15 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+func RegisterStatuserMetrics() {
+	prometheus.MustRegister(TotalSentAvailabilityCheckReqs, AvailabilityCheckReqsDuration, TotalInvalidAvailabilityCheckReqs)
+}
+
+func RegisterApiMetrics() {
+	// no metrics
+}
+
+func RegisterWorkerMetrics() {
+	prometheus.MustRegister(JobQueueSize, JobQueueInFlight, BackgroundJobDuration, ReservationCount)
+}
```

Comment:
Moving registration functions outside of the main file, it is starting to be a bit messy.